knowhere/knowhere/utils/CMakeLists.txt

```diff
@@ -45,3 +45,5 @@ add_library(knowhere_utils STATIC
 
 target_compile_options(knowhere_utils PUBLIC "-msse4.2")
 target_link_libraries(knowhere_utils PUBLIC openblas)
+
+target_include_directories(knowhere_utils PUBLIC ${INDEX_SOURCE_DIR}/knowhere/knowhere/utils)
```

Comment:
redundant

Reply:
This is necessary for external reference